Loaded module: cuda/12.2
/dtu/p1/johlau/Thesis-Synthex/.venv/lib/python3.9/site-packages/keras/src/layers/layer.py:360: UserWarning: `build()` was called on layer 'patch_encoder', however the layer does not have a `build()` method implemented and it looks like it has unbuilt state. This will cause the layer to be marked as built, despite not being actually built, which may cause failures down the line. Make sure to implement a proper `build()` method.
  warnings.warn(
0it [00:00, ?it/s]0it [00:00, ?it/s]
Traceback (most recent call last):
  File "/dtu/p1/johlau/Thesis-Synthex/.venv/lib/python3.9/site-packages/PIL/Image.py", line 3130, in fromarray
    mode, rawmode = _fromarray_typemap[typekey]
KeyError: ((1, 1, 3), '<f4')

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/dtu/p1/johlau/Thesis-Synthex/object_detection/crop_padchest.py", line 51, in <module>
    image = Image.fromarray(image_array)
  File "/dtu/p1/johlau/Thesis-Synthex/.venv/lib/python3.9/site-packages/PIL/Image.py", line 3134, in fromarray
    raise TypeError(msg) from e
TypeError: Cannot handle this data type: (1, 1, 3), <f4
